src/App/Router.jsx

```diff
@@ -2,13 +2,15 @@ import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import DashBoardPage from '../pages/DashBoard';
 import LandingPage from '../pages/LandingPage';
 import RegisterPage from '../pages/RegisterPage';
+import LoginPage from '../pages/LoginPage';
 
 function Router() {
   return (
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<LandingPage />} />
         <Route path="/register" element={<RegisterPage />} />
+        <Route path="/login" element={<LoginPage />} />
```

Comment:
rewrite this line and follow best practice

Reply:
already follow la